tests/utils/0030_df.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Verify zenfs df command works as expected.
+
+$ZENFS_DIR/zenfs df --zbd=$ZDEV >> $TEST_OUT
+RES=$?
+if [ $RES -ne 0 ]; then
+  exit $RES
+fi
+
+LINES=$(cat $TEST_OUT | wc -l)
```

Comment:
This just checks that the output is four lines, can't we come up with something that would provide better test coverage?